docker/jenkins/Makefile

```diff
@@ -2,13 +2,22 @@ BASE_FILE=docker-compose.yml
 OVERRIDE_FILE=docker-compose.override.yml
 SERVICE=ci
 SETUP_SCRIPT=../nvidia/setup_nvidia_docker_compose.py
+ENV_VARIABLES=CUDA_VERSION GPU_ARCH
 
 all: setup_nvidia_docker
 
 setup_nvidia_docker: ${OVERRIDE_FILE}
 
-${OVERRIDE_FILE}: ${BASE_FILE} ${SETUP_SCRIPT} Makefile
+${OVERRIDE_FILE}: ${BASE_FILE} ${SETUP_SCRIPT} Makefile ${ENV_VARIABLES}
 	${SETUP_SCRIPT} --extended-file ${BASE_FILE} --extend-services ${SERVICE} --override-file ${OVERRIDE_FILE}
 
 clean:
-	rm -vf *_generated ${OVERRIDE_FILE}
+	rm -vf *_generated ${OVERRIDE_FILE} ${ENV_VARIABLES}
```

Comment:
So this will reset the values of the new environment variables?

Reply:
No this is just a convenience rule to remove all the files that are generated.

Reply:
ok - because the variables are put into a file

Reply:
Correct.  And you can inspect the values in the [workspace for that build on Jenkins](https://cloud.cees.ornl.gov/jenkins-ci/job/DataTransferKit-experimental/compiler=GCC54,slave=fetnat01/ws/docker/jenkins/)